app/services/inherited_proofing/va/service_payload_validator.rb

```diff
@@ -0,0 +1,74 @@
+require_relative 'field_validatable'
+
+module InheritedProofing
+  module Va
+    # Validates the payload_hash returned from the inherited proofing service.
+    class ServicePayloadValidator
+      include FieldValidatable
+
+      # TODO NOW: regex need to be verified and/or added
+      FIELD_TEMPLATE = {
+        first_name: { regex: /[a-zA-Z\s]/, required: true },
```

Comment:
What is the thinking behind the regexes in these validations? It is not safe to assume that names will consist entirely of characters from the 26 letter latin alphabet.

Reply:
Yes, ty, that was WIP. I am working on using POSIX bracket expressions now.